windows/windows_update_install/windows_update_install.yml

```diff
@@ -30,10 +32,23 @@
         - name: "Install the online updates for guest OS"
           include_tasks: install_online_updates.yml
 
-        - name: "Prepare the .msu file"
+        - name: "Prepare the .msu file of type {{ msu_file_type }}"
```

Comment:
Where did you set `msu_file_type` used at here and below tasks names?

Reply:
Will modify the task name without using the var.